docs/dbpedia-spotlight-wrapper/v1.0/metadata.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Dbpedia Spotlight Wrapper",
+  "description": "Apply named entity linking to all text documents in a MMIF file.",
+  "app_version": "v1.0",
+  "mmif_version": "1.0.0",
+  "analyzer_version": "version_1.0",
+  "app_license": "Apache 2.0",
+  "analyzer_license": "Apache 2.0",
+  "identifier": "http://apps.clams.ai/dbpedia-spotlight-wrapper/v1.0",
+  "url": "https://github.com/clamsproject/app-dbpedia-spotlight-wrapper",
+  "input": [
+    {
+      "@type": "http://mmif.clams.ai/vocabulary/TextDocument/v1",
+      "required": true
+    }
+  ],
+  "output": [
+    {
+      "@type": "http://vocab.lappsgrid.org/NamedEntity"
+    }
+  ],
+  "parameters": [
+    {
+      "name": "confidence",
+      "description": "disambiguation confidence score for linking",
+      "type": "number",
+      "default": 0.5,
+      "multivalued": false
+    },
+    {
+      "name": "support",
+      "description": "resource prominence, i.e. number of in-links in Wikipedia (lower bound)",
+      "type": "integer",
+      "default": 0,
+      "multivalued": false
+    },
+    {
+      "name": "types",
+      "description": "types filter",
+      "type": "string",
+      "multivalued": false
```

Comment:
`types` parameter has a description that's not necessarily very helpful. But from the name of it I think it can be used to limit the recognition or linking algorithm to certain types of named entity. If that's the case, can't it be `multivalued`? 

* just in case @wricketts decide to change it to a multivalued parameter, this will require a small change in the `_annotate` as well. 
> Multivalued: (optional, False by default) Set True if the parameter can have multiple values.Note that, for parameters that allow multiple values, the SDK will pass a singleton list to `_annotate()` even when one value is passed via HTTP.
(https://sdk.clams.ai/appmetadata.html)